jsonschema/tests/fuzz_validate.py

```diff
@@ -0,0 +1,39 @@
+import sys
+
+from hypothesis import given
+from hypothesis import strategies as st
+import atheris
+
+import jsonschema
+
+PRIM = st.one_of(
+    st.booleans(),
+    st.integers(min_value=-(2 ** 63), max_value=2 ** 63 - 1),
+    st.floats(allow_nan=False, allow_infinity=False),
+    st.text(),
+)
+DICT = st.recursive(
+    base=st.dictionaries(st.text(), PRIM),
```

Comment:
```suggestion
    base=st.booleans() | st.dictionaries(st.text(), PRIM),
```

Booleans are also valid schemas (!), and that's been a fertile source of bugs for my schema-processing logic especially variously nested.

Reply:
Cheers!